### ISSUE TITLE
SDV in Pyspark 

### ISSUE THREAD

**Issue by kvrameshreddy** [sdv-dev/SDV#176]
* SDV version:0.3.6
* Python version:3.7
* Operating System:Windows 8.1

### Description
how to run SDV model in Spark session. Is there any work around to run SDV in pyspark

### What I Did

![pyspark error](https://user-images.githubusercontent.com/49742709/89280400-3573a100-d666-11ea-8891-6e8ec1961b89.JPG)

**Comment by yairdata** [comment]
@kvrameshreddy  interesting, want to try it also.
according to the code it looks like you should pass a list or dict to the table and not spark dataframe.
try converting it first to a list and see if it helps

**Comment by JagdishKolhe** [comment]
I think, SDV and the libraries used underneath the SDV  might not support the PySpark

**Comment by Baukebrenninkmeijer** [comment]
Indeed, there is currently no support for pyspark. Since SDV is built on several sub-libraries that approach this problem differently, that support should be implemented per sub-library.

**Comment by npatki** [comment]
Hi, we have an open issue for PySpark support in #573 that we're using for tracking. I'll resolve this issue in favor of that one. 

Please feel to continue contributing to the open issue.
